backend/src/main/java/pt/ulisboa/tecnico/socialsoftware/tutor/user/User.java

```diff
@@ -75,9 +76,12 @@
     @OneToMany(cascade = CascadeType.ALL, mappedBy = "user", fetch = FetchType.LAZY, orphanRemoval=true)
     private Set<QuizAnswer> quizAnswers = new HashSet<>();
 
-    @ManyToMany
+    @ManyToMany(fetch = FetchType.EAGER)
```

Comment:
This cannot be!!! It is a many to many relation, it has a huge performance impact